uast/uast_test.go

```diff
@@ -217,3 +217,27 @@ func TestOrderTokens(t *testing.T) {
 		"id3", "Prefix+", "tok_pre_left", "subright_in1", "tok_pre_right", "subright_in2"}
 	require.Equal(expected, result)
 }
+
+func TestCyclomaticComplexity(t *testing.T) {
+	require := require.New(t)
+	n := &Node{InternalType: "empty",
```

Comment:
I am sorry, I am a little bit out of shape on this, can you explain why an "empty" node is required? (no need to explain that on the code, just in this thread will be enough)